int32 dir_actuator1
int32 dir_actuator2
float32 pwm_actuator1
float32 pwm_actuator2
int32 dir_bevel1
int32 dir_bevel2
float32 pwm_bevel1
float32 pwm_bevel2
int32 gripper 
float32 grip_opener 
int32 dir_base
float32 pwm_base

